waterdata/templates/monitoring_location.html

```diff
@@ -6,6 +6,16 @@
 {% block extra_head_tags %}
     {% if status_code == 200 %}
         <meta name="description" content="{{ components.Description(stations[0].site_no, location_with_values, parm_grp_summary) }}">
+        <!-- tags for Facebook Open Graph -->
+        <meta property="og:url" content="{{ url_for('monitoring_location', site_no=site_no, _external=True) }}" />
+        <meta property="og:type" content="website" />
+        <meta property="og:title" content="{{ page_title }}" />
```

Comment:
Does this work? I'm a little surprised because it's not set until line 31. If it is working, maybe move line 31 before this part of the code.